### DISCUSSION TITLE
Week 6 Impressions:

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#120] [General]
# Video/Mihai:
The video and Mihai’s experience were good reminders of the struggle that many people must work through daily. In medicine, I am exposed to a lot of patients with disabilities and have the opportunity to discuss how it affects their life. What I loved is the depiction of Mihai and how much of an impact he had on others but also on what he worked on. Often, when we think of disability we think of incapability. This story shows how incorrect this is. Often, those with some disability have abilities beyond what we may perceive as normal which have great benefit to society. 

# FreeCodeCamp:
FreeCodeCamp was good but I feel like I didn’t learn much from it. I think I need to go over these assignments again to remember how to create the color patterns and as well as accessibility stuff. I will definitely use the scroll function in my site.

# Project Thoughts:
- **Blog** & **CV/Personal Statement** & **Contact form**
  - Blog: My biggest concerns is if I should have one large page, or a small Table of Contents with links to specific articles I write. Something I can do is create dropdown tabs that open the article on the same page via Java which will be a bit difficult, or I can link the table of contents to that article within the page. My preference is the dropdown tab for asthetics.
  - CV/Personal Statement: Since my CV is long and might require a lot of CSS to organize data, I might have some difficulties with CSS. I plan on creating a specific CSS page for the CV but also a general CSS for the header/footer and general aesthetics. I also might do the similar dropdown menu for each category of my CV. 
  - Contact form: My biggest issue is the email function. I did read that was out of the scope of this class to have it email me directly, so I am not going to worry about this issue. 
  - **I will be creating a nutrition website, but I don’t plan on including it in this project as it will take time to create the articles and information. I have a few nutrition articles I created that I will post to the blog and will need to update my personal statement and CV in order to start that portion.**

## Mock Site New Format
<img width="1091" alt="Screenshot 2023-06-12 at 12 45 48 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132800749/02a1098c-1a5f-46b7-8f73-1ff8b3d37aea">

**Comment by coopsand** [comment]
I think you are onto a great start with your site!

**Comment by metasean** [comment]
***I absolutely love your summary of the video & Mihai’s experience!***

I think you're going to run into some accessibility issues with your header's color scheme, you can check it using an accessibility contrast checker, such as https://webaim.org/resources/contrastchecker/

**Reply by derekscholes** [reply]
Thanks for this link! It will definitely help with my color decisions. I do agree that it is slightly difficult to read because of the contrast between colors. I will definitely also use this in other situations like presentations since I never really knew about this before!
